.gitignore

```diff
@@ -14,5 +14,9 @@ docs/overrides/
 # python files
 __pycache__
 
+# zig files
+codes/zig/zig-cache
```

Comment:
Please create a `.ignore` file in the `codes/zig` dir

Reply:
> Please create a `.ignore` file in the `codes/zig` dir

OK. I've already done it.